package.json

```diff
@@ -66,7 +66,7 @@
   },
   "dependencies": {
     "@dotlottie/react-player": "^1.6.3",
-    "@expensify/react-native-live-markdown": "0.1.143",
+    "@expensify/react-native-live-markdown": "0.1.163",
```

Comment:
Since the inline images were added in version `0.1.145,` you can only bump it to `0.1.144`

Reply:
Well, I need at least `0.1.146` cause it has web fixes for this pr...